service/conf/routes

```diff
@@ -29,6 +29,7 @@ POST  /v1/user/unblock			      @controllers.usermanagement.UserStatusController.
 
 GET   /v1/role/read						    @controllers.usermanagement.UserRoleController.getRoles()
 POST  /v1/user/assign/role			  @controllers.usermanagement.UserRoleController.assignRoles()
+POST  /private/v1/user/assign/role	  @controllers.usermanagement.UserRoleController.assignRoles()
```

Comment:
use /private/user/v1/assign/role - same as private api for get user by externalId